source/common/options.js

```diff
@@ -149,12 +149,23 @@ function buildOptionsPage() {
 
   settings.forEach(function (setting) {
     if (setting.type === 'checkbox') {
-      jq('#' + setting.section + 'SettingsPage').append(jq('<div>', { class: 'row option-row' }).append(jq('<input>', { type: 'checkbox', id: setting.name, name: setting.name, 'aria-describedby': setting.name + 'HelpBlock' })).append(jq('<div>', { class: 'option-description' }).append(jq('<label>', { for: setting.name, text: setting.title })).append(jq('<span>', { id: setting.name + 'HelpBlock', class: 'help-block', text: setting.description }))));
+      jq('#' + setting.section + 'SettingsPage')
+        .append(jq('<div>', { class: 'row option-row' })
+          .append(jq('<input>', { type: 'checkbox', id: setting.name, name: setting.name, 'aria-describedby': setting.name + 'HelpBlock' }))
+          .append(jq('<div>', { class: 'option-description' })
+            .append(jq('<label>', { for: setting.name, text: setting.title }))
+            .append(jq('<span>', { id: setting.name + 'HelpBlock', class: 'help-block' }))));
+      jq('#' + setting.name + 'HelpBlock').html(setting.description);
```

Comment:
Are they going to accept something like this?
Add this to options.html:
`<script type="text/javascript" src="marked.js"></script>`
then in options.js:
`
let markDown = marked(setting:description);`
followed by
`
jq('#' + setting.name + 'HelpBlock').html(markDown);
`
In the end, the marked down text still has to be rendered as html and the html is not going to be static. I have updated things as shown above and it does work but I wonder if they're going to whine about this approach as well.

Coding the markdown in settings.json file is pretty much a PIA. I think we should put it in a separate file. Because markdown relies on a certain amount of formatting (i.e. new lines) a line in the settings.json file could easily exceed 500 bytes. Maybe it's possible to allow newlines, spaces, and tabs in the settings.json file? Thoughts?